src/components/banner/styles/Banner.scss

```diff
@@ -0,0 +1,92 @@
+@import '~coveo-styleguide/scss/common/palette.scss';
+@import '~coveo-styleguide/scss/variables.scss';
+
+.banner {
+    background: linear-gradient(45deg, $mirage 0%, $science-blue 100%);
+    box-shadow: $banner-box-shadow;
+    padding: $banner-padding;
+    height: $banner-height;
+    box-sizing: border-box;
+    position: relative;
+
+    button {
+        background: $picton-blue;
+        color: $pure-white;
+        display: inline-block;
+        border-radius: $border-radius;
+        text-transform: uppercase;
+        padding: $button-padding-y $button-padding-x;
+        border: none;
+        font-size: $button-font-size;
+        cursor: pointer;
+
+        &:hover {
+            opacity: $transparency-2;
+        }
+
+        &:global(.primary) {
+            background-color: $orange;
+            border: none;
+        }
+    }
+}
+
+.bannerName {
```

Comment:
use a - for classes : banner-name

Reply:
or you use this naming because its more simple to access your variable inside the component ? like styles.bannerWarningTitle

Reply:
You can only use camelCase in CSS Modules